configs/defs/generic_linux.textproto

```diff
@@ -9558,3 +9558,406 @@ benchmark_configs: {
+  compliance_note: {
+    title: "Ensure sshd GSSAPIAuthentication is disabled"
+    description:
+      "The GSSAPIAuthentication parameter specifies whether user authentication "
+      "based on GSSAPI is allowed. Disabling this option reduces the attack "
+      "surface of the system by avoiding exposure of the system's GSSAPI to remote hosts."
+    rationale:
+      "Allowing GSSAPI authentication through SSH exposes the system's GSSAPI to remote "
+      "hosts, increasing the attack surface. Disabling it minimizes potential risks."
+    remediation:
+      "Edit the /etc/ssh/sshd_config file and set the following option:\n"
+      "```\n"
+      "GSSAPIAuthentication no\n"
+      "```\n"
+      "Ensure this is placed above any Include or Match entries."
+    cis_benchmark: {
+      profile_level: 1
+      severity: LOW
+    }
+    scan_instructions:
+      "generic:{"
+      "  check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{single_file:{path:\"/etc/ssh/sshd_config\"}}"
+      "      content_entry:{"
+      "        match_type: ALL_MATCH_ANY_ORDER"
+      "        match_criteria:{"
+      "          filter_regex: \"\\\\s*GSSAPIAuthentication\\\\s+.*$\""
+      "          expected_regex: \"\\\\s*GSSAPIAuthentication\\\\s+no$\""
+      "        }"
+      "      }"
+      "    }"
+      "  }"
+      "}"
+  }
+}
+
+
+
+benchmark_configs: {
+  id: "sshd-hostbasedauthentication-disabled"
+  compliance_note: {
+    title: "Ensure SSH HostbasedAuthentication is disabled"
+    description:
+      "The HostbasedAuthentication parameter specifies if authentication is allowed through "
+      "trusted hosts via the use of .rhosts or /etc/hosts.equiv, along with successful public "
+      "key client host authentication. This option only applies to SSH Protocol Version 2."
+    rationale:
+      "Even though the .rhosts files are ineffective if support is disabled in /etc/pam.conf, "
+      "disabling the ability to use .rhosts files in SSH provides an additional layer of protection."
+    remediation:
+      "Edit the /etc/ssh/sshd_config file to set the HostbasedAuthentication parameter to no "
+      "above any Include and Match entries as follows:\n"
+      "```\n"
+      "HostbasedAuthentication no\n"
+      "```\n"
+      "Note: First occurrence of an option takes precedence. If Include locations are enabled, "
+      "used, and order of precedence is understood in your environment, the entry may be created "
+      "in a file in an Include location."
+    cis_benchmark: {
+      profile_level: 1
+      severity: LOW
+    }
+    scan_instructions:
+      "generic:{"
+      "  check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{single_file:{path:\"/etc/ssh/sshd_config\"}}"
+      "      content_entry:{"
+      "        match_type: ALL_MATCH_ANY_ORDER"
+      "        match_criteria: {"
+      "          filter_regex: \"\\\\s*HostbasedAuthentication\\\\s+.*$\""
```

Comment:
I believe we automatically include ^ and $ to the filter+expected regexes so there's no need to add them separately.
https://github.com/google/localtoast/blob/16cb3b302fa5155317c449c8d6a25b98ace56e91/scannerlib/configchecks/content_entry_file_check.go#L86